packages/graphback-codegen-resolvers/src/plugins/ResolverGeneratorPlugin.ts

```diff
@@ -0,0 +1,210 @@
+
+        const outputResolvers = {};
+        for (const { graphqlType } of models) {
+            const modelResolvers = generatedResolvers[graphqlType.name];
+
+            const queries = this.createCustomResolvers(graphqlType, queryType, Object.keys(modelResolvers.Query));
+            const mutations = this.createCustomResolvers(graphqlType, mutationType, Object.keys(modelResolvers.Mutation));
+            const subscriptions = this.createCustomSubscriptionResolvers(graphqlType, subscriptionType, Object.keys(modelResolvers.Subscription));
+
+            const typeResolvers = {
+                Query: queries,
+                Mutation: mutations,
+                Subscription: subscriptions
+            }
+
+            outputResolvers[graphqlType.name] = typeResolvers;
+        }
+
+        return outputResolvers;
+    }
+
+    private createMutations(graphqlType: GraphQLObjectType, crudOptions: GraphbackCRUDGeneratorConfig) {
+        const mutations = {};
+
+        const objectName = graphqlType.name.toLowerCase();
+        if (crudOptions.create) {
+            const fieldName = getFieldName(graphqlType.name, GraphbackOperationType.CREATE);
+            // tslint:disable-next-line: no-any
+            mutations[fieldName] = createTemplate(objectName, crudOptions.subCreate)
+        }
+        if (crudOptions.update) {
+            const fieldName = getFieldName(graphqlType.name, GraphbackOperationType.UPDATE);
+            mutations[fieldName] = updateTemplate(objectName, crudOptions.subUpdate);
+        }
+        if (crudOptions.update) {
+            const fieldName = getFieldName(graphqlType.name, GraphbackOperationType.UPDATE);
+            mutations[fieldName] = updateTemplate(objectName, crudOptions.update);
+        }
+        if (crudOptions.delete) {
+            const fieldName = getFieldName(graphqlType.name, GraphbackOperationType.DELETE);
+            mutations[fieldName] = updateTemplate(objectName, crudOptions.delete);
+        }
+
+        return mutations;
+    }
+
+    private createQueries(graphqlType: GraphQLObjectType, crudOptions: GraphbackCRUDGeneratorConfig) {
+        const queries = {};
+        const objectName = graphqlType.name.toLowerCase();
+
+        if (crudOptions.find) {
+            const fieldName = getFieldName(objectName, GraphbackOperationType.FIND, 's');
+            queries[fieldName] = findTemplate(objectName);
+        }
+        if (crudOptions.findAll) {
+            const fieldName = getFieldName(objectName, GraphbackOperationType.FIND_ALL, 's');
+            queries[fieldName] = findAllTemplate(objectName);
+        }
+
+        return queries;
+    }
+
+    private createSubscriptions(graphqlType: GraphQLObjectType, crudOptions: GraphbackCRUDGeneratorConfig) {
+        const objectName = graphqlType.name.toLowerCase();
+
+        const subscriptions = {};
+        if (crudOptions.create && crudOptions.subCreate) {
+            const fieldName = `new${graphqlType.name}`;
```

Comment:
We need CRUD name format comming from core. Needs todo to use core helper yo get method name

Reply:
Current subscription resolvers don't use helper to get method name:

https://github.com/aerogear/graphback/blob/e7d8c75679925489f165562facb62fa8f2f18260/packages/graphback-codegen-resolvers/src/templates/LayeredResolverTemplates.ts#L43

Reply:
Yes. But then we have this contract between schema resolvers and client not written anywhere. I have added issue to build our own  GraphQLCRUD spec.